README.md

```diff
@@ -20,41 +20,41 @@ Build Swift for Android from your Mac
 
 ## Usage
 
-### In Android Studio
+### In Android Studio / Gradle
 
-In your project’s `CMakeLists.txt`, add the following code after the `cmake_minimum_required(VERSION x.y.z)` declaration.
+You probably want Gradle to compile your native sources automatically when building the android app. In order to archieve this, you have to create another `CMakeLists.txt` file and reference it from `app/build.gradle`.
 
+```
+externalNativeBuild {
+    cmake {
+        version "3.16.2"
+        path "CMakeLists.txt" // android/app/src/CMakeLists.txt
```

Comment:
I'm pretty sure we don't need this, because it should be the default @michaelknoch

Reply:
seems to be required, otherwise cmake is not invoked at all